app/models/submission.rb

```diff
@@ -371,6 +376,12 @@ def aud
     assessment.aud_for course_user_datum_id
   end
 
+  def update_cud_grade_watchlist_instances_if_latest
+    if self.aud.latest_submission_id == self.id
+      self.course_user_datum.update_cud_grade_watchlist_instances
```

Comment:
Would this mean that when one submission is tweaked, all the watchlist instances is updated?